Do not register `const float *` type on utiliy_ops.cu

malfet opened the pull request:
Summary: But rather use `float *`, which is alredy registered

Test Plan: CI

Differential Revision: D19221405

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 47478e17:


* **1/1** broken upstream at merge base 218cfd56 (see [grid view](https://dr.pytorch.org/master-timeline.html?count=1&sha1=218cfd568da4dffaa0129acfc3c3f33c5f0bd0d4&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35))
  <details>
  <summary>
  You may want to rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch origin viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  
* **0/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=47478e17653b2ab78963b90a298e6e3c1d46a9e8).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably caused by upstream breakages:
* [pytorch_linux_xenial_py3_clang5_asan_test](https://dr.pytorch.org/build-details.html?build_id=78591626)



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

malfet committed:
Do not register `const float *` type on utiliy_ops.cu (#31583)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/31583

But rather use `float *`, which is alredy registered

Test Plan: CI

Differential Revision: D19221405

fbshipit-source-id: 4269fea3bb08c19211fb505f55da9b2cb9ff3c84
